crates/sui-protocol-config/src/lib.rs

```diff
@@ -3917,6 +3927,10 @@ impl ProtocolConfig {
                 91 => {
                     cfg.feature_flags.per_command_shared_object_transfer_rules = true;
                 }
+                92 => {
```

Comment:
91 is not yet present in any release branch so you can still safely add to it rather than creating a new version.